src/app/scripts/cac/urlrouting/cac-urlrouting.js

```diff
@@ -14,8 +14,8 @@ CAC.UrlRouting.UrlRouter = (function (_, $, UserPreferences, Utils, Navigo) {
     var SHARED_READ = ['maxWalk', 'wheelchair', 'bikeTriangle'];
     var EXPLORE_ENCODE = SHARED_ENCODE.concat(['placeId', 'exploreTime']);
     var EXPLORE_READ = EXPLORE_ENCODE.concat(SHARED_READ);
-    var DIRECTIONS_ENCODE = SHARED_ENCODE.concat(['destination', 'destinationText', 'waypoints']);
-    var DIRECTIONS_READ = DIRECTIONS_ENCODE.concat(SHARED_READ).concat(['arriveBy']);
+    var DIRECTIONS_ENCODE = SHARED_ENCODE.concat(['destination', 'destinationText']);
+    var DIRECTIONS_READ = DIRECTIONS_ENCODE.concat(SHARED_READ).concat(['arriveBy', 'waypoints']);
```

Comment:
So this means the address bar URL omits waypoints, right?  So to share an edited route you'd need to use the share button?  Right now the shared itinerary ignores waypoints, I think because this is looking for `waypoints` but the parameter being included in the URL is `intermediatePlaces`.

But yeah, also this means that if you make an edited itinerary such that you have waypoints in your local storage then you follow a link copy/pasted from the address bar, you end up with the start/end/etc parameters encoded in the link but routing through the waypoints you defined for your old route.  If we can't include waypoints in the address bar URL, we need to at least figure out how to clear them when appropriate.